src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +128,20 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void helpTest() {
+        UI.events.registerEvent((NavigateToPageEventHandler) e -> url = e.url);
+        getFirstPanelField();
+        push(KeyCode.F1);
+        sleep(1000);
```

Comment:
We can't know that this will take exactly one second. We should wait until the condition is true using `waitAndAssertEquals`.